### ISSUE TITLE
Add a function to export a text list of "Loader Mods"

### ISSUE THREAD

**Issue by Targren** [MultiMC/MultiMC5#4150]
### Role

Support/Assistance 

### Suggestion

Be able to export a Text list of mods installed in an instance

### Benefit

Screenshots of mod folders are miserable to use when trying to help someone troubleshoot

### This suggestion is unique

- [X] I have searched the issue tracker and did not find an issue describing my suggestion, especially not one that has been rejected.

**Comment by Targren** [comment]
Are you sure they're included in the game logs by default, or just when an exception occurs?

**Comment by Zetabite** [comment]
they are default for mmc logs, which are more usefull than the usual mc crash log

**Comment by Zetabite** [comment]
The file logs are shit for most stuff. If you want a proper log, let them run the game once with the issue. If the issue is consistent, it will appear again, else its not reproducible and mods will probably dont do much anyway then.

**Comment by Targren** [comment]
> The file logs are shit for most stuff.

Yeah, that was my thought when you originally said "have them send the log". Now I'm on the same page.

**Comment by izfaruqi** [comment]
Hello, I'm currently looking for a good first issue to contribute to. Can I take on this issue?

**Comment by Zetabite** [comment]
just do it, rarely anyone contributes

**Comment by izfaruqi** [comment]
Umm, I see. Okay then.
